lms/djangoapps/static_template_view/tests/test_views.py

```diff
@@ -26,6 +32,64 @@ def test_about(self):
         """
         self._test_view('about', 'text/html')
 
+    def test_about_with_site_configuration(self):
+        """
+        Test the about view with the header and content set in SiteConfiguration.
+        """
+        test_header = "Very Unique Test Header"
+        test_content = "Very Unique Test Content"
+        test_header_key = 'static_template_about_header'
+        test_content_key = 'static_template_about_content'
+        response = None
+        configuration = {test_header_key: test_header, test_content_key: test_content}
+        with with_site_configuration_context(configuration=configuration):
+            response = self.client.get(reverse("about"))
+        self.assertIn(test_header, response.content)
+        self.assertIn(test_content, response.content)
+
+    def test_about_with_site_configuration_and_html_in_header(self):
+        """
+        Test the about view with html in the header.
+        """
+        test_header = "<i>Very Unique Test Header</i>"
+        test_content = "Very Unique Test Content"
+        test_header_key = 'static_template_about_header'
+        test_content_key = 'static_template_about_content'
+        response = None
+        configuration = {test_header_key: test_header, test_content_key: test_content}
+        with with_site_configuration_context(configuration=configuration):
+            response = self.client.get(reverse("about"))
+        self.assertIn(test_content, response.content)
```

Comment:
Why aren't you checking that the header is in the content?  The test is about the header.  Also, seems like this test and the next one could be combined.